src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -112,14 +127,23 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow one single parameterless annotation on the same line
+ * Example to allow multiple annotations on the same line:
```

Comment:
this config looks like allow everything ? does this allow everything ?

Reply:
Yes. I couldn't find an instance where it violates.

Reply:
so lets be honest with user ... `Example to allow any location of annotations:`

Reply:
Done